chainerx_cc/chainerx/backward_builder.h

```diff
@@ -136,7 +136,7 @@ class BackwardBuilder {
     ~BackwardBuilder() { CHAINERX_ASSERT(is_finalized_); }
 
     BackwardBuilder(const BackwardBuilder&) = delete;
-    BackwardBuilder(BackwardBuilder&&) noexcept = default;
+    BackwardBuilder(BackwardBuilder&&) = default;
```

Comment:
Why is this needed?
`noexcept` was requested by clang-tidy: https://clang.llvm.org/extra/clang-tidy/checks/performance-noexcept-move-constructor.html

Reply:
needed by d15fcad, no longer for now